blueflood-core/src/main/java/com/rackspacecloud/blueflood/service/ScheduleContext.java

```diff
@@ -214,22 +214,12 @@ public void update(long millis, int shard) {
      * already running or scheduled, then add them to the queue of scheduled
      * slots.
      *
-     * Eligibility:
-     * 1) slots should be older than {@code maxAgeMillis}
-     * 2) If slot is identified as being picked for re-roll, slots should be
-     *    older than {@code delayedMetricsMaxAgeMillis}
+     * Note that {@code maxAgeMillis}, {@code rollupDelayForMetricsWithShortDelay}
+     * {@code rollupWaitForMetricsWithLongDelay} are age values, not a timestamp.
      *
-     * Note that both {@code maxAgeMillis} and {@code delayedMetricsMaxAgeMillis}
-     * are age values, not a timestamp.
-     * If the difference between {@link #scheduleTime} and a given slot's
-     * {@link UpdateStamp} in milliseconds is greater than
-     * {@code maxAgeMillis}, then that slot will scheduled.
-     *
-     * @param maxAgeMillis
-     * @param delayedMetricsMaxAgeMillis
      */
     // only one thread should be calling in this puppy.
-    void scheduleEligibleSlots(long maxAgeMillis, long delayedMetricsMaxAgeMillis) {
+    void scheduleEligibleSlots(long maxAgeMillis, long rollupDelayForMetricsWithShortDelay, long rollupWaitForMetricsWithLongDelay) {
```

Comment:
Would you please add brief description of these 3 parameters.  At the every least I was confused with maxAgeMillis since that term isn't used elsewhere (that I noticed).  It's rollupDelayMillis in other places, right?